dockerfiles/s3-registry/index.ts

```diff
@@ -4,17 +4,27 @@ import dynamico from '@dynamico/express-middleware';
 import { S3Storage } from '@dynamico/s3-storage';
 const { S3 } = require('aws-sdk');
 import cors from 'cors';
+import nconf from 'nconf';
```

Comment:
Adding `nconf` is a good idea. Can you change other usages or `process.env.whatever` to `nconf.get`?

Reply:
Sure, no problem 👍